Rubberduck.Core/UI/Inspections/InspectionResultsViewModel.cs

```diff
@@ -116,13 +116,14 @@ private void _configService_SettingsChanged(object sender, ConfigurationChangedE
             _runInspectionsOnReparse = e.RunInspectionsOnReparse;
         }
 
-        private ObservableCollection<IInspectionResult> _results = new ObservableCollection<IInspectionResult>();
+        private ObservableCollection<IInspectionResult> _resultsAll = new ObservableCollection<IInspectionResult>();
+        private ObservableCollection<IInspectionResult> _resultsGroupedAndFiltered = new ObservableCollection<IInspectionResult>();
```

Comment:
This worries the SQL boy in me. 😆 

Question: is this workable even when you're doing with large number (say, >1000) inspections? Does the time for filtering/grouping increase linearly with number of inspection results?

Reply:
I don't believe the filtering has any issues. Even when I had some results with ~700 filter and group was an immediate response. It too longer for the inspections to resolve than it did for them to filter.